src/fiatconnect/index.ts

```diff
@@ -146,20 +145,17 @@ export async function getFiatConnectQuotes(
     provider: fiatConnectProviders.find((provider) => provider.id === result.id)!,
   }))
 }
-export type FetchQuotesInput = Omit<QuotesInput, 'fiatConnectProviders'> & {
+export type FetchQuotesInput = QuotesInput & {
   fiatConnectCashInEnabled: boolean
   fiatConnectCashOutEnabled: boolean
-  account: string
 }
 
 export async function fetchQuotes(params: FetchQuotesInput) {
-  const { account, fiatConnectCashInEnabled, fiatConnectCashOutEnabled, ...quotesInput } = params
+  const { fiatConnectCashInEnabled, fiatConnectCashOutEnabled, ...quotesInput } = params
```

Comment:
nit: use destructuring assignment for `flow` too?

Reply:
Its cleaner to not destructure IMO because it needs to be passed with quotesInput

Reply:
Ah, missed that this is also needed in quotesInput